vm/chef/cookbooks/kafka/files/kafka

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /opt/c2d/c2d-utils
+
+readonly kafka_nodes="$(get_attribute_value "kafka-nodes")"
+readonly kafka_nodes_count="$(echo "${kafka_nodes}" | wc -w)"
+readonly kafka_config="/opt/kafka/config/server.properties"
+readonly zookeeper_config="/opt/kafka/config/zookeeper.properties"
+
+# Configure Kafka in Cluster mode
+if [[ "${kafka_nodes_count}" -gt 1 ]]; then
```

Comment:
Can we add tests for multicluster setup? (Either here or as part of DM tests)

Reply:
Sure. We are adding at DM tests as there we can have multiple instances.